packages/editor/src/components/properties/TransformPropertyGroup.tsx

```diff
@@ -82,6 +82,7 @@ export const TransformPropertyGroup: EditorComponentType = (props) => {
   const onChangePosition = (value: Vector3) => {
     const nodes = getEntityNodeArrayFromEntities(getMutableState(SelectionState).selectedEntities.value)
     EditorControlFunctions.positionObject(nodes, [value])
+    LocalTransformComponent.stateMap[props.entity]!.set(LocalTransformComponent.valueMap[props.entity])
```

Comment:
These changes feel a little hacky. I know we keep valueMap in sync with stateMap through a subscribe. I found that as stock. If we set the UseEffect in collider to have a child of localtransform.scale that is being marked for rerendering. But I wonder why stateMap as a whole isn't being marked for rerendering when we change one of its members through updateComponent but it does work with your methodology.

Reply:
Yeah the problem here is we aren't setting things reactively at all, just the underlying bitecs threejs proxy objects.